### PR TITLE
Amending docs for KillSwitch #20708

### PR DESCRIPTION
**Pull request by svezfaz**
Refs #20708

As agreed: I have removed the Scala code from being showed in the Java doc page.

Is there anything else needed for this ticket? 
The Java sample is actually in place, do you think it needs improvements?

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

File: akka-docs/rst/scala/stream/stream-dynamic.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ The ``KillSwitch`` trait allows to complete or fail the graph(s).
 .. includecode:: ../../../../akka-stream/src/main/scala/akka/stream/KillSwitch.scala
    :include: kill-switch
 
-After the first call to either ``shutdown`` and ``abort``, all subsequent calls to any of these methods will be ignored.
```
Comment (ktoso) [inline akka-docs/rst/scala/stream/stream-dynamic.rst]: is the same typo present in the java version of the doc? if so please fix there too :)

**Comment by ktoso** [comment]
LGTM once https://github.com/akka/akka/pull/20716#discussion_r65779450 answered

File: akka-docs/rst/scala/stream/stream-dynamic.rst (hunk 1 above)
Comment (svezfaz) [inline akka-docs/rst/scala/stream/stream-dynamic.rst]: Yes, it's fixed there as well

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM, thanks!

**Pull request merged**